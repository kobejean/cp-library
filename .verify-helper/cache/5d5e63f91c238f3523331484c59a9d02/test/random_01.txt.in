99
7484303 1989474 125660016
615372777 119787692 462224593
728078051 635851437 576823355
596594455 423750036 954291757
144200759 100896705 893229302
974352699 380534861 37119022
828189871 267327757 807497925
542974003 19718539 613363438
879194483 411418038 992120464
50192403 7519404 736386595
803753191 124480368 441504628
175964554 28720769 599715811
273978301 219335692 209550382
110149676 106348642 936333553
613751993 52359007 394002980
66679008 25998794 414068747
40760521 38182090 675811768
519691750 8702686 741452977
790699289 164389539 860076763
5962758 3043915 997981495
932516293 408191174 28415297
520874243 422916664 587344338
959709973 125091359 434654931
757058049 519264704 129816176
891260017 844108039 283891837
843241502 708531464 22451673
139596997 44912905 396941015
421261360 30794932 281450237
837843529 728175010 478539926
20622967 8319337 994309826
839677661 159285514 3142445
770896447 207995476 119183522
71088517 28540647 890192054
509213004 16426787 601655395
771967873 707052981 731661785
213175647 185705261 545698665
859604474 703194552 600753696
517384562 465880140 657850948
381299769 346464098 503356284
77040494 15401226 142420898
644491297 336669892 648921164
496255570 189950267 642176095
664019550 5267292 603414714
643557396 176005048 22509536
456767997 90410761 450595359
498458102 99585624 178994182
638885014 514133780 55996994
406466962 309830733 572976322
93630191 9439644 315903565
279819720 42243190 881702350
693146496 259068312 753614300
310889843 215392859 1326347
90944983 2085956 729645820
409894282 95763649 639409370
276213322 145930564 436119528
637340890 610335106 628338962
446555650 206820296 690899586
665371103 145941341 236960173
297581732 137462938 95610094
702283599 179254620 821772777
7577169 329565 890786811
763605410 385637230 54725840
178214285 109451918 814774625
628150053 367340724 358296900
92212846 29994822 229684136
924115457 643694655 489116122
798506550 595791300 504134475
957767405 158738720 81424660
569295929 408835940 569295929
577051954 136764605 577051954
358884937 30186941 358884937
793975550 56525134 793975550
209741449 19865439 629224347
218360236 99135920 436720472
704158907 328538097 704158907
369440765 229418098 369440765
684370417 231039143 684370417
715110994 422001313 715110994
999999963 658536561 536585346
1000000000 400000000 100000000
999999963 73170729 48780486
999999963 902438991 926829234
999999990 866666658 477777773
999999990 266666664 588888883
999999980 114285712 385714278
999999910 230769210 373626340
999999938 755101994 479591807
999999990 911111102 688888882
999999962 967213078 754098332
999999990 534246570 273972600
999999988 230769228 192307690
999999999 333333333 666666666
999999990 696969690 515151510
999999980 826086940 608695640
999999976 323529404 382352932
999999982 868421037 921052615
999999980 434782600 239130430
999999962 983606520 639344238
999999972 981481454 18518518
