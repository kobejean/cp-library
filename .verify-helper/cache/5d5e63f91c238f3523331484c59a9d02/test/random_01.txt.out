3808352
49
728077983
153346772
53
974352610
94733182
100
879194456
41628783
7
175964485
19956644
62
613751929
28089026
57
519691682
616963529
23
932516213
319504738
24
757058045
293436356
6
139596916
233416764
44
20622923
121656876
50
71088499
93018487
75
-1
16
258692274
-1
74
2199553
-1
22
160889284
-1
44
319442475
-1
96
27981971
70854486
72
882960
-1
82
318670442
178269139
5
148790849
216558303
80
76360520
-1
13
4191474
-1
40
191553432
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
23
6
19
26
84
42
36
77
86
19
8
9
4
1
20
20
7
11
40
36
1
